target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -38,6 +38,9 @@ mediatek_setup_interfaces()
 	routerich,ax3000-ubootmod)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" wan
 		;;
+	cudy,wr3000h-v1)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" wan
```

Comment:
This is the same as the default ( `*) ucidef_... ;;` ) and can be removed.

Reply:
You are right, is not needed for this version of the router. New push.